pkg/kubelet/images/image_manager.go

```diff
@@ -85,7 +87,15 @@ func (m *imageManager) EnsureImageExists(pod *api.Pod, container *api.Container,
 		glog.Errorf("Couldn't make a ref to pod %v, container %v: '%v'", pod.Name, container.Name, err)
 	}
 
-	spec := kubecontainer.ImageSpec{Image: container.Image}
+	// If the image contains no tag or digest, a default tag should be applied.
+	image, err := applyDefaultImageTag(container.Image)
```

Comment:
> Yeah, just don't know whether different runtimes have the same default. For now, it's fine because everyone is using docker image, and the default should all be :latest (?).

If that's true, why did we even try to apply the default tag in dockertools? :-)